sdk/storage/azure-storage-files-shares/src/share_file_client.cpp

```diff
@@ -991,7 +991,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
     {
       if (options.HttpHeaders.ContentHash.Algorithm != HashAlgorithm::Md5)
       {
-        abort();
+        std::abort();
```

Comment:
Can you help me understand, what's the difference here between abort and std::abort?

Reply:
It's just a pattern thing. `abort()` is more C style and `std::abort()` is more C++ style.